ChangeLog.md

```diff
@@ -42,13 +42,15 @@ Current Trunk
 - Use LLVM's new pass manager by default, as LLVM does. This changes a bunch of
   things about how LLVM optimizes and inlines, so it may cause noticeable
   changes in compile times, code size, and speed, either for better or for
-  worse. (#13427)
+  worse. You can use the old pass manager (until LLVM removes it) by passing
+  `-flegacy-pass-manager` (#13427)
```

Comment:
Maybe mention the link flag too?

Reply:
How could that be used? `--lto-legacy-pass-manager` needs to be passed to `wasm-ld` but the user doesn't run it directly. Do we have a way to pass flags to it? (I don't see one.)

Reply:
Yes `-Wl,--foo` is the way to pass flags to the linker.

Reply:
Also, sadly even if the user passed both compiler and linker flags they won't effect the standard libraries (which have a fixed set of flags they build with).

Reply:
Oh, does `emcc` parse and pass those to `wasm-ld`?

Reply:
Yup.. except for a few that we don't support and want to silently ignore (`UNSUPPORTED_LLD_FLAGS`)